lib/modules/dosomething/dosomething_home/dosomething_home.module

```diff
@@ -38,7 +38,13 @@ function dosomething_home_preprocess_node(&$vars) {
       // Grab all tiles attributes.
       $tiles = array();
       $tiles['nid'] = $nid;
-      $tiles['is_staff_pick'] = $node->field_staff_pick[$lang_code][0]['value'];
+
+      $display_staff_pick = FALSE;
+      if (data_get($node->field_staff_pick, $lang_code) && data_get($node->field_staff_pick_label, $lang_code)) {
```

Comment:
We don't have to live in the dark ages any longer: https://github.com/DoSomething/phoenix/pull/6702